<launch>
    <node pkg="rosbag" type="play" name="bag_player" 
          args="--clock $(find fsd_common_msgs)/bag/lidar_cone_side_slam_state.bag" 
          output="screen" required="true"/>
    <node pkg="fsd_common_msgs" type="subscriber.py" name="cone_subscriber" 
          output="screen" required="true"/>
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find fsd_common_msgs)/rviz/cone_visualization.rviz" />
</launch>
